src/components/Table/GridTable.test.tsx

```diff
@@ -137,6 +138,36 @@ describe("GridTable", () => {
     expect(cell(r, 2, 0)).toHaveTextContent("2");
   });
 
+  it("wraps readonly select field", async () => {
```

Comment:
Suggestion: Really minor but maybe call this "does not truncate reaodnly select fields"?

I get it is effectively wrapping, and it's called `wrap` in the code, but I think this name more matches what we're asserting against.

Reply:
cool, it seems better, thanks 👏

Reply:
This also addresses it for any field, not just SelectFields. Thanks to updating it in `TextFieldBase`, we get the fix in DateField, NumberField, TextField, SelectField, MultiSelectField, etc... 🎉